webui/react/src/pages/TrialDetails/TrialDetailsHyperparameters.tsx

```diff
@@ -0,0 +1,54 @@
+import { ColumnType } from 'antd/es/table';
+import React, { useMemo } from 'react';
+
+import ResponsiveTable from 'components/ResponsiveTable';
+import { ExperimentBase, TrialDetails } from 'types';
+import { isObject } from 'utils/data';
+import { alphanumericSorter } from 'utils/sort';
+
+export interface Props {
+  experiment: ExperimentBase;
+  trial: TrialDetails;
+}
+
+interface HyperParameter {
+  hyperparameter: string,
+  value: string,
+}
+
+const TrialDetailsHyperparameters: React.FC<Props> = ({ trial }: Props) => {
+  const columns: ColumnType<HyperParameter>[] = useMemo(() => [
+    {
+      dataIndex: 'hyperparameter',
+      defaultSortOrder: 'ascend',
+      key: 'hyperparameter',
+      sorter: (a: HyperParameter, b: HyperParameter) =>
+        alphanumericSorter(a.hyperparameter, b.hyperparameter),
+      title: 'Hyperparameter',
+    },
+    {
+      dataIndex: 'value',
+      key: 'value',
+      title: 'Value',
+    },
+  ], []);
```

Comment:
**super nit:** this is a total personal preference thing so feel free to ignore. A space after this line even though it's variable declarations, just because it's a series of large `useMemo` blocks. So using a space to break these blocks up a bit. Similar to how you was organize `useCallbacks`.